src/middleware/axios-error-handler.js

```diff
@@ -0,0 +1,18 @@
+const { AxiosError } = require('axios')
+const { HttpError } = require('../index')
+
+/**
+ * Express middleware error handler for converting any axios-thrown errors to HTTP Errors
+ *
+ * @param {Error | import('axios').AxiosError} err
+ * @param {import('express').Request} req: express request object
+ * @param {import('express').Response} res: express response object
+ * @param {import('express').NextFunction} next: express next
+ */
```

Comment:
```suggestion
 * @type {import('express').ErrorRequestHandler}
 */
```
Although if we do this, we're not explicitly saying `AxiosError` is an expected type of `err`. If we care about that, we can use a custom global type definition which extends Express, similar to how we've done it in `portal-utils`